Exercise_B/index.html

```diff
@@ -1,34 +1,62 @@
 <!DOCTYPE html>
 <html>
-  <head>
-    <meta charset="utf-8" />
-    <title>Flower Shop BS</title>
-    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
-    <!--If you want to use bootstrap properly this is vital (helps browser to measure the page and readjust the information accordingly-->
 
-    <!-- Link and CDN required for bootstrap -->
+<head>
+  <meta charset="utf-8" />
+  <title>Flower Shop BS</title>
+  <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+  <!--If you want to use bootstrap properly this is vital (helps browser to measure the page and readjust the information accordingly-->
 
-    <!-- Google Font - Lato -->
-    <link
-      href="https://fonts.googleapis.com/css?family=Lato"
-      rel="stylesheet"
-    />
-    <!-- Custom Stylesheet -->
-    <link rel="stylesheet" href="./assets/style/stylesheet.css" />
-  </head>
+  <!-- Link and CDN required for bootstrap -->
+  <link href="https://cdn.jsdelivr.net/npm/bootstrap@5.1.3/dist/css/bootstrap.min.css" rel="stylesheet"
+    integrity="sha384-1BmE4kWBq78iYhFldvKuhfTAU6auU8tT94WrHftjDbrCEXSU1oBoqyl2QvZ6jIW3" crossorigin="anonymous" />
+  <!-- Google Font - Lato -->
+  <link href="https://fonts.googleapis.com/css?family=Lato" rel="stylesheet" />
+  <!-- Custom Stylesheet -->
+  <link rel="stylesheet" href="./assets/style/stylesheet.css" />
+</head>
 
-  <body>
-    <!--Header (flower shop logo block element)-->
-    <!--Banner Jumbotron of the image-->
-
-    <!--Four containers with img / text-->
-    <!-- 4 containers -->
-
-    <!-- Payments Div -->
+<body>
+  <!--Header (flower shop logo block element)-->
+  <header>
+    <img src="/Exercise_B/assets/images/logo.png" alt="">
+  </header>
+  <!--Banner Jumbotron of the image-->
+  <section class="d-flex flex-column justify-content-center align-items-center text-white"
+    style="background-image: url(/Exercise_B/assets/images/flowershop.jpg); height: 70vh;">

```

Comment:
For this link, you could use a CSS file to add in the background image to add in the banner at the back. Or you could add in a container and then add the image inside and that could work too. Jumbotron is deprecated with the new Bootstrap 5.0.